sdk/pom.xml

```diff
@@ -455,7 +455,7 @@
     <dependency>
```

Comment:
This now becomes a dependency with `test` scope, right?

I wonder why strict enforcement of dependencies didn't uncover this. @kennknowles, any ideas?